active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+    <Image Source="Assets/LargeParisMap.png"/>
+</ScrollViewer>
+```
+
+## Controlling the Minimum and Maximum Zoom Factor
+Use the `MinZoomFactor` and `MaxZoomFactor` properties to control the amount the user may zoom the content. These properties are ignored if the ZoomMode is Disabled.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView Width="500" Height="400" ContentOrientation="Both"
+                   ZoomMode="Enabled" 
+                   MinZoomFactor="1.0" MaxZoomFactor="2.0">
+    <Image Source="Assets/LargeParisMap.png"/>
+</mux:ScrollingView>
+```
+
+*ScrollViewer (existing)*
+
+```xml
+<ScrollViewer Width="500" Height="400" HorizontalScrollBarVisibility="Auto"
+              ZoomMode="Enabled" MinZoomFactor="1.0" MaxZoomFactor="2.0">
+    <Image Source="Assets/LargeParisMap.png"/>
+</ScrollViewer>
+```
+
+## Controlling the user's ability to scroll
+
+The `ScrollingView`'s HorizontalScrollMode and VerticalScrollMode properties affect the user interactivity of the control. They do not affect its layout.
+
+HorizontalScrollMode determines whether the user can interact with the control with touch, mouse wheel, scroll bar, etc. to scroll the content horizontally. Similarly VerticalScrollMode is for the vertical direction. The content can still be scrolled programatically though.
+
+The possible values are Enabled and Disabled, and the default value for both directions is Enabled.
+
+When the scroll mode is Enabled and the content is smaller than the viewport, the user can still wiggle (overpan) the content with touch input.
+
+Changes to the HorizontalScrollMode and VerticalScrollMode properties take effect immediately and may affect an on-going user interaction.
+
+This example shows vertical-only scrolling. Even though the content is wider than the viewport, the user cannot scroll horizontally.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView x:Name="ScrollingView"
+                  HorizontalScrollMode="Disabled" VerticalScrollMode="Enabled"
+                  ContentOrientation="Both">
+    <Grid ...>
+    </Grid>
+</mux:ScrollingView>
+```
+
+This example shows horizontal-only scrolling. Even if the content is taller than the viewport, the user cannot scroll vertically.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView x:Name="ScrollingView"
+                  HorizontalScrollMode="Enabled" VerticalScrollMode="Disabled"
+                  ContentOrientation="Both">
+    <Grid ...>
+    </Grid>
+</mux:ScrollingView>
+```
+
+## Turning off built-in MouseWheel support
+
+In this example the mouse wheel is disabled from scrolling the `ScrollingView`. This allows the PointerWheelChanged events to be processed to respond to mouse-wheel input in a custom way while still leaving Pen and Touch inputs active for the `ScrollingView`.
+
+*`ScrollingView` (new)*
+
+```xml
+<mux:ScrollingView ContentOrientation="Both" IgnoredInputKind="MouseWheel">
```

Comment:
Is this a collection? Can I ignore both Pen & Wheel together?  How are other accessibility devices represented?

Reply:
This is a flag enum with the list of input types the ScrollingView knows about.

Reply:
The property and type should be plural since it's a [flags].